### PR TITLE
Added a new test for automationexcersise.com

### PR DESCRIPTION
**Pull request by Zoltekk**
Added a new test

### CODE REVIEW

File: automationexcersise/conftest.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    driver.get(get_settings["base_url"])
+
+    assert WebDriverWait(driver, 5).until(EC.presence_of_element_located((By.XPATH, get_xpaths["home"]["home_button"])))
+    login_button = driver.find_element(By.XPATH, get_xpaths["login"]["login_subpage"])
+    login_button.click()
+
+    assert WebDriverWait(driver, 5).until(EC.presence_of_element_located((By.XPATH, "//div[@class='signup-form']")))
+    signup_name = driver.find_element(By.XPATH, "//input[@data-qa='signup-name']")
+    signup_email = driver.find_element(By.XPATH, "//input[@data-qa='signup-email']")
+    signup_button = driver.find_element(By.XPATH, "//button[@data-qa='signup-button']")
```
Comment (Glovacky) [inline automationexcersise/conftest.py]: Remember what I told you about "stale" elements. It's best to get the element right before interacting with it.

File: automationexcersise/conftest.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    assert WebDriverWait(driver, 5).until(
+        EC.presence_of_element_located((By.XPATH, "//*[contains(text(), 'Enter Account "
+                                                  "Information')]")))
+    title = driver.find_element(By.XPATH, "//input[@id='id_gender1']")
+    password = driver.find_element(By.XPATH, "//input[@id='password']")
+    days = driver.find_element(By.XPATH, "//select[@id='days']/option[@value='1']")
+    months = driver.find_element(By.XPATH, "//select[@id='months']/option[@value='1']")
+    years = driver.find_element(By.XPATH, "//select[@id='years']/option[@value='2000']")
+    newsletter = driver.find_element(By.XPATH, "//input[@id='newsletter']")
+    optin = driver.find_element(By.XPATH, "//input[@id='optin']")
```
Comment (Glovacky) [inline automationexcersise/conftest.py]: same here. This will probably generate some stale elements

File: automationexcersise/conftest.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    company = driver.find_element(By.XPATH, "//input[@id='company']")
+    address1 = driver.find_element(By.XPATH, "//input[@id='address1']")
+    address2 = driver.find_element(By.XPATH, "//input[@id='address2']")
+    close_ad = driver.find_element(By.XPATH, "//div[@class='grippy-host']")
+    country = driver.find_element(By.XPATH, "//select[@id='country']/option[@value='United States']")
+    state = driver.find_element(By.XPATH, "//input[@id='state']")
+    city = driver.find_element(By.XPATH, "//input[@id='city']")
+    zipcode = driver.find_element(By.XPATH, "//input[@id='zipcode']")
+    mobile = driver.find_element(By.XPATH, "//input[@id='mobile_number']")
+    create_account = driver.find_element(By.XPATH, "//button[@data-qa='create-account']")
```
Comment (Glovacky) [inline automationexcersise/conftest.py]: same

File: task1/test_task_2.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    driver = get_driver
+    driver.get(get_settings["base_url"])
+
+    login = driver.find_element(By.XPATH, get_xpaths["login"]["username_input"])
+    password = driver.find_element(By.XPATH, get_xpaths["login"]["password_input"])
+    login_button = driver.find_element(By.XPATH, get_xpaths["login"]["login_button"])
+
+    login.send_keys(username)
+    password.send_keys(passwd)
+    login_button.click()
```
Comment (Glovacky) [inline task1/test_task_2.py]: do you want stale elements? because that's how you get stale elements

File: automationexcersise/conftest.py (hunk 1 above)
Comment (Zoltekk) [reply automationexcersise/conftest.py]: Fixed

File: automationexcersise/conftest.py (hunk 3 above)
Comment (Zoltekk) [reply automationexcersise/conftest.py]: Fixed

File: task1/test_task_2.py (hunk 4 above)
Comment (Zoltekk) [reply task1/test_task_2.py]: Fixed

File: automationexcersise/conftest.py (hunk 2 above)
Comment (Zoltekk) [reply automationexcersise/conftest.py]: Fixed

**Pull request merged**